tests/test-cross-context-dispatch/ccd-ee8-webapp/src/main/java/org/eclipse/jetty/tests/ccd/ee8/CCDServlet.java

```diff
@@ -0,0 +1,85 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.tests.ccd.ee8;
+
+import java.io.IOException;
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.eclipse.jetty.tests.ccd.common.DispatchPlan;
+import org.eclipse.jetty.tests.ccd.common.Step;
+import org.eclipse.jetty.tests.ccd.common.steps.ContextRedispatchStep;
+import org.eclipse.jetty.tests.ccd.common.steps.GetHttpSessionStep;
+import org.eclipse.jetty.tests.ccd.common.steps.RequestDispatchStep;
+
+public class CCDServlet extends HttpServlet
```

Comment:
It's possible to use the `ee9 -> ee8` translation maven plugin to auto generate the classes, might want to consider that.

Reply:
No benefit for that.
First, this ccd-ee8-webapp is actually different than the ccd-ee9-webapp.
Also, there's no jetty-ee8 dependencies in this webapp for the translation plugin to work against.
It's a webapp, with jetty-core deps.

Reply:
This can easily be translated from the `ccd-ee9-webapp` to replace `jakarta.servlet.` with `javax.servlet`. Also I think both the `ee8` and `ee9` tests should do the same object wrapper identity test, so the code should be identical anyway.